cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -0,0 +1,136 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.rest.gen_data import users as _users
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.rest import assert_response
+from cfme.utils.wait import wait_for_decorator
+
+pytestmark = [test_requirements.report, pytest.mark.tier(3), pytest.mark.sauce]
+
+
+@pytest.fixture(scope="function")
+def rbac_api(appliance, request):
+    user, user_data = _users(
+        request, appliance, password="smartvm", group="EvmGroup-user"
+    )
+    yield appliance.new_rest_api_instance(
+        entry_point=appliance.rest_api._entry_point,
+        auth=(user.userid, user_data["password"]),
+    )
+    appliance.rest_api.collections.users.action.delete(id=user.id)
+
+
+@test_requirements.rest
+@pytest.mark.tier(1)
+def test_non_admin_user_reports_access_rest(appliance, request, rbac_api):
+    """ This test checks if a non-admin user with proper privileges can access all reports via API.
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/12h
+        tags: report, rest
+        setup:
```

Comment:
Is it fine adding two tags?
I think, it will create confusion while collecting under particular focus areas.